src/fixture.ts

```diff
@@ -67,17 +105,26 @@ const parsePytestOutputToFixtures = (output: string) => {
  */
 export const getFixtures = (document: vscode.TextDocument) => {
     let response;
-    const args = ["--fixtures", document.uri.fsPath];
+    const args = ["--fixtures", "-v", document.uri.fsPath];
+    const cwd = vscode.workspace.workspaceFolders?.[0]?.uri.fsPath;
+    if(!cwd)
```

Comment:
`vscode.workspace.workspaceFolders` is empty only if the user opens a single file (instead of a folder or a workspace). Will check if we can avoid using cwd.